Descrtiption: Don't pass -DOPJ_STATIC when compiling the OpenJPEG code, as it
 makes the symbols hidden when including /usr/include/openjpeg-2.1/openjpeg.h
 .
 Fixes a FTBFS against libopenjp2-7 (>= 2.1.1)
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Bug-Debian: https://bugs.debian.org/832873

--- a/base/lib.mak
+++ b/base/lib.mak
@@ -1830,11 +1830,11 @@
 
 $(GLOBJ)sjpx_openjpeg.$(OBJ) : $(GLSRC)sjpx_openjpeg.c $(AK) \
  $(memory__h) $(gserror_h) $(gserrors_h) \
  $(gdebug_h) $(strimpl_h) $(sjpx_openjpeg_h) $(LIB_MAK) $(MAKEDIRS)
 	$(GLJPXOPJCC) $(GLO_)sjpx_openjpeg.$(OBJ) \
-		$(C_) -DOPJ_STATIC $(GLSRC)sjpx_openjpeg.c
+		$(C_) $(GLSRC)sjpx_openjpeg.c
 
 # ---------------- Pixel-difference filters ---------------- #
 # The Predictor facility of the LZW and Flate filters uses these.
 
 pdiff_=$(GLOBJ)spdiff.$(OBJ)
